feconf.py

```diff
@@ -51,9 +51,13 @@
 TESTS_DATA_DIR = os.path.join('core', 'tests', 'data')
 SAMPLE_EXPLORATIONS_DIR = os.path.join('data', 'explorations')
 SAMPLE_COLLECTIONS_DIR = os.path.join('data', 'collections')
-INTERACTIONS_DIR = os.path.join('extensions', 'interactions')
-GADGETS_DIR = os.path.join('extensions', 'gadgets')
-RTE_EXTENSIONS_DIR = os.path.join('extensions', 'rich_text_components')
+
+EXTENSIONS_DIR_PREFIX = 'build_staging' if (IS_MINIFIED or not DEV_MODE) else ''
```

Comment:
Also I think I'm slightly confused. Is the plan to upload build_staging/ to production and serve from there? My conception was that it is meant to be a temporary directory of sorts that's used only within the build process, but if so it shouldn't be given this sort of status within feconf. The prepending of build_staging should only happen in build.py.